public/app/features/logs/components/LogDetailsRow.tsx

```diff
@@ -93,29 +93,42 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
     if (onClickFilterLabel) {
       onClickFilterLabel(parsedKey, parsedValue);
     }
+
+    reportInteraction('grafana_explore_logs_log_details_filter_clicked', {
+      datasourceType: this.props.row.datasourceType,
```

Comment:
Sorry, I think I could have been more clear. Destructuring here would also be great 😊